web/public/js/directives/cd-anchor/index.js

```diff
@@ -0,0 +1,23 @@
+;(function() {
```

Comment:
Minor thing, but we use 2 spaces for indentation. Since you need to fix the other comment, it'd be nice if you could fix the indentation here too.